createdisk.sh

```diff
@@ -127,7 +127,7 @@ if [ "${ARCH}" == "aarch64" ] && [ ${BUNDLE_TYPE} != "okd" ]; then
    # Install qemu-user-static-x86 package from fedora koji to run x86 image on M1
    # Not supported by RHEL https://access.redhat.com/solutions/5654221 and not included
    # in any subscription repo.
-   ${SSH} core@${VM_IP} -- "sudo rpm-ostree install https://kojipkgs.fedoraproject.org//packages/qemu/8.2.4/1.fc40/aarch64/qemu-user-static-x86-8.2.4-1.fc40.aarch64.rpm"
+   ${SSH} core@${VM_IP} -- "sudo rpm-ostree install https://kojipkgs.fedoraproject.org//packages/qemu/8.2.6/4.fc40/aarch64/qemu-user-static-x86-8.2.6-4.fc40.aarch64.rpm"
```

Comment:
8.2.6-4 has no tags (https://koji.fedoraproject.org/koji/buildinfo?buildID=2547717), the build could go away as well. Let's use https://koji.fedoraproject.org/koji/buildinfo?buildID=2535777 which is the latest released update for f40.